.github/workflows/ci-tests.yml

```
@@ -0,0 +1,99 @@
+# Reference:
+#   - https://github.com/actions/cache
+#   - https://github.com/actions/checkout
+#   - https://github.com/conda-incubator/setup-miniconda
+
+name: ci-tests
+
+on: [pull_request, push]
```

```suggestion
on:
  pull_request:

  push:
    branches-ignore:
      - "conda-lock-auto-update"
      - "pre-commit-ci-update-config"
      - "dependabot/*"
```

See bjlittle/geovista#50.